src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Data/ListCollectionView.cs

```diff
@@ -2930,7 +2930,7 @@ private void SetSortDescriptions(SortDescriptionCollection descriptions)
         // SortDescription was added/removed, refresh CollectionView
         private void SortDescriptionsChanged(object sender, NotifyCollectionChangedEventArgs e)
         {
-            if (IsAddingNew || IsEditingItem)
+            if ((IsAddingNew || IsEditingItem) && e.Action != NotifyCollectionChangedAction.Reset)
```

Comment:
Could you please provide a descriptive comment about the condition being checked here?